packages/docs-theme/src/components/typescript/apiHeader.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import { isTsClass, isTsInterface, ITsDocBase } from "documentalist/dist/client";
+import * as React from "react";
+import { DocumentationContextTypes, IDocumentationContext } from "../../common/context";
+
+export class ApiHeader extends React.PureComponent<ITsDocBase> {
+    public static contextTypes = DocumentationContextTypes;
+    public static displayName = "Docs.ApiHeader";
+
+    public context: IDocumentationContext;
+
+    public render() {
+        return (
+            <div className="docs-interface-header">
+                <div className="docs-interface-name">
+                    <small>{this.props.kind}</small> {this.props.name} <small>{this.renderInheritance()}</small>
+                </div>
+                <small className="docs-package-name">
+                    <a href={this.props.sourceUrl} target="_blank">
+                        @blueprintjs/{this.props.fileName.split("/", 2)[1]}
```

Comment:
@adidahiya thoughts on how to make this configurable? perhaps a new context function `getPackageName(entry: ITsDocBase)` and some prop on `Documentation`?

Reply:
or `renderViewSourceLinkText(entry: ITsDocBase): string` which defaults to simply returning "View source"

Reply:
yeah I like the latter approach. default to "View source" and don't do anything too fancy with package paths in this docs-theme package (we can be fancier in docs-app).